main.go

```diff
@@ -108,7 +110,7 @@ func DecodedSignal(line string) (stop bool) {
 		}
 
 		switch device {
-		case GT_WT_01:
+		case GtWt01:
```

Comment:
I thought you wanted to add a new device for "GT-WT-01 variant", and not to overwrite the existing one?

Reply:
Just renamed it for the linter. The problem is, that the detection only uses the pulse length and not the actual values. Therefore, both are GT_WT_01 but fields are detected at runtime.